build.sbt

```diff
@@ -55,9 +55,12 @@ val commonSettings = Seq(
       </developers>
 )
 
+lazy val root = project.in(file("."))
+  .aggregate(`gremlin-scala`, macros)
+
```

Comment:
Is this useful?
Please read "Default root project" :
http://www.scala-sbt.org/0.13/tutorial/Multi-Project.html